src/cloud-api-adaptor/ibmcloud/image/import.sh

```diff
@@ -101,7 +103,9 @@ if [ -z "$operating_system" ]; then
     image_arch="${image_name##*-}"
     operating_system="ubuntu-20-04-${image_arch/x86_64/amd64}"
 fi
-image_name="$(echo ${image_name,,} | sed 's/\./-/g' | sed 's/_/-/g')"
+# image_name="$(echo ${image_name,,} | sed 's/\./-/g' | sed 's/_/-/g')"
+image_name="$(echo ${image_name} | tr '[:upper:]' '[:lower:]' | sed 's/\./-/g' | sed 's/_/-/g')"
+# ^^^ the ,, parameter expansion is not available on mac, so use tr command instead
```

Comment:
Same here

Reply:
Sure, I can remove the comments if you think that's best. My logic for adding them was to prevent someone from changing it to the "better" way, not realizing we need to do it this way for portability to Mac.